assets/pages/contributions-landing/newContributionsLanding.jsx

```diff
@@ -74,12 +75,59 @@ const countryGroupSpecificDetails: {
   },
 };
 
+const selectedCountry = countryGroups[countryGroupId];
 
 // ----- Render ----- //
 
 const content = (
   <Provider store={store}>
-    <Page header={<header />} footer={<footer />}>
+    <Page
+      header={
+        <header role="banner" className="gu-content__header">
+          <a className="glogo" href="https://www.theguardian.com"><img src="/assets-payment/roundel.svg" alt="The Guardian" /></a>
+          <details className="countries">
+            <summary aria-label="Selected country: {selectedCountry.name} ({currencies[selectedCountry.currency]})">
```

Comment:
This is producing a string verbatim in the DOM. To fix this you can do the following:

```jsx
<summary aria-label={`Selected country: ${selectedCountry.name} (${currencies[selectedCountry.currency].glyph})`>
```

I've also assumed you wanted the `glyph` property here, because `currencies[selectedCountry.currency]` returns an object?

Reply:
😮 I would never have seen that. Thanks!